src/agent/GRPCClientAgent.ts

```diff
@@ -72,20 +74,32 @@ class GRPCClientAgent extends GRPCClient<AgentServiceClient> {
   }
 
   @ready(new grpcErrors.ErrorGRPCClientNotStarted())
-  public vaultsGitInfoGet(...args) {
+  public vaultsGitInfoGet(
```

Comment:
I think there's a robust solution possibly here: https://github.com/MatrixAI/js-polykey/issues/200. Should investigate that here.

Reply:
Note that the https://gitlab.com/MatrixAI/Engineering/Polykey/js-polykey/-/merge_requests/213 has moved the grpc types into `grpc/types.ts` and it also has `*Client` versions of those stream objects.

Reply:
Noted.